src/pages/IstioConfigList/__tests__/IstioConfigListComponent.test.ts

```diff
@@ -97,7 +111,7 @@ describe('IstioConfigListComponent#toIstioItems', () => {
     const istioItems = toIstioItems(unfiltered);
 
     expect(istioItems).toBeDefined();
-    expect(istioItems.length).toBe(33);
+    expect(istioItems.length).toBe(42);
```

Comment:
42 ... yeah of course, how could it be different? :)

Reply:
What do you mean?

Reply:
nothing wrong https://es.wikipedia.org/wiki/El_sentido_de_la_vida,_el_universo_y_todo_lo_dem%C3%A1s

Reply:
i feel old, again

Reply:
Oh, I see! I have that pending, friends recommend me the movie :rocket: 
Similar thing happens with number 23, they even did a movie for that (way worse that yours though)